pyvisa/attributes.py

```diff
@@ -2611,7 +2611,7 @@ class AttrVI_ATTR_WIN_SIZE(RangeAttribute):
 
     visa_name = "VI_ATTR_WIN_SIZE"
 
-    visa_type = "ViBusSize64" if constants.is_64bits else "ViBusSize"
```

Comment:
The ViBusSize64 type exists according to the VISA specs so I would prefer to add it in types.py rather than using a type differing from the specs even if they all just alias one another.

Reply:
Fixed.